core/pkg/resultshandling/results.go

```diff
@@ -65,13 +65,16 @@ func (resultsHandler *ResultsHandler) GetResults() *reporthandlingv2.PostureRepo
 // HandleResults handle the scan results according to the pre defined interfaces
 func (resultsHandler *ResultsHandler) HandleResults() error {
 
-	resultsHandler.printerObj.ActionPrint(resultsHandler.scanData)
-
 	if err := resultsHandler.reporterObj.Submit(resultsHandler.scanData); err != nil {
 		return err
```

Comment:
issue: printing results should happen before submits

This is a change that, interestingly enough, broke support for SARIF output and exposed one dirty hack we do. I will be fixing this myself.